src/app/page.module.css

```diff
@@ -1,6 +1,110 @@
 .main {
-  display: grid;
-  place-items: center;
-  padding: 6rem;
-  min-height: 100vh;
+    display: grid;
+    place-items: center;
+    align-content: center;
+    padding: 1rem;
+    min-height: 100vh;
+}
+
+/* Makes the title rotate on hover */
+.title {
+    font-size: 3rem;
+    text-transform: uppercase;
+    transform: rotate(0deg);
+    transition: transform 1.2s;
+    padding: 10vw;
+    cursor: default;
+}
+
+.title:hover {
+    transform: rotate(360deg);
+}
+
+/* Makes the blob pulse on hover */
+.blob {
+    --blob-color: 236, 59, 59;
```

Comment:
Would be nice to see this definition along with `--clr-green-rgb`

Reply:
Create a new `--clr-red-rgb` in the root and change this line to `--blog-color: var(--clr-red-rg)` 😉